### DISCUSSION TITLE
How to prevent non-mapped tables from being dropped during schema update ?

### DISCUSSION THREAD

**Discussion by mpecro** [laravel-doctrine/orm#649] [General]
Hello,

I have several tables that are not mapped with Doctrine (technical tables).

When I dump the SQL from schema:update, I see that Doctrine wants to drop these non-mapped tables.

Thank you.

**Comment by TomHAnderson** [comment]
This is not an issue with this library.  Research `schema_filter` https://www.doctrine-project.org/projects/doctrine-migrations/en/3.8/reference/generating-migrations.html for more information
